apps/src/templates/GameButtons.jsx

```diff
@@ -1,48 +1,107 @@
-var msg = require('../locale');
+import msg from '../locale';
 
-var ProtectedStatefulDiv = require('./ProtectedStatefulDiv');
-var commonStyles = require('../commonStyles');
+import ProtectedStatefulDiv from './ProtectedStatefulDiv';
+import commonStyles from '../commonStyles';
+import experiments from '../experiments';
+import classNames from 'classnames';
+import Radium from 'radium';
+import { connect } from 'react-redux';
 
-var styles = {
+const styles = {
   instructionsInTopPane: {
     // common.scss provides an :after selector that ends up adding 18px of height
     // to gameButtons. We want to get rid of that when we have don't have
     // instructions below game buttons
     marginBottom: -18
+  },
+  runImage: {
+    width: 26,
+    height: 26,
+    background: 'url("/blockly/media/common_images/shared-sprites-26x26.png") 0 0'
+  },
+  resetImage: {
+    width: 26,
+    height: 26,
+    background: 'url("/blockly/media/common_images/shared-sprites-26x26.png") 0 26px'
   }
 };
 
+export const RunButton = Radium(props => (
+  <button
+      id="runButton"
+      className={classNames(['launch', 'blocklyLaunch', props.hidden && 'invisible'])}
+      style={props.style}
+  >
+    <div>
+      {msg.runProgram()}
+    </div>
+    <img
+        src="/blockly/media/1x1.gif"
+        style={[!props.isMinecraft && styles.runImage]}
+        className={classNames([props.isMinecraft && "run26"])}
```

Comment:
Don't love this. Minecraft has different styles for these than everyone else, and those would be somewhat more of a pain to inline, so we use classes for MC and style elsewhere (tho for other parts of this same component we use classes everywhere).

Another option would be to create a separate MinecraftGameButtons which may or may not be better.